testing/test_aux.out

```diff
@@ -68,9 +70,9 @@ RESULT info communication.type.broadcast Broadcast packets received. Unicast pac
 RESULT skip connection.network.ntp_support No NTP packets received.
 RESULT skip connection.network.ntp_update Not enough NTP packets received.
 RESULT pass connection.mac_oui Manufacturer: Google found for address 3c:5a:b4:1e:8f:0a
-dhcp requests 1 1 1 1
+dhcp requests 1 1 1 0
 3c5ab41e8f0a: []
-3c5ab41e8f0b: ['3c5ab41e8f0b:ping:TimeoutError']
+3c5ab41e8f0b: ['3c5ab41e8f0b:ipaddr:TimeoutError', '3c5ab41e8f0b:ping:TimeoutError']
```

Comment:
likewise. Is this new timeout expected and/or why?

Reply:
@grafnu The timeout was intentional. The ipaddr timeout for that faux device is set to 1 (to always timeout instantly) as that device is the intended fail-coverage device (sends no DHCP requests). Therefore there is no reason to have the ipaddr code running for 2000s if we know the outcome is going to be no DHCP requests anyway. An alternative would be to disable ipaddr on the second faux device and to check the existence of the test_ipaddr.pcap file. However, this should probably cause a skip case rather than a fail.